guides/migrate-from-redshift.md

```diff
@@ -0,0 +1,191 @@
+        ,likeclassical BOOLEAN   -- ENCODE RAW
+        ,likeopera BOOLEAN   -- ENCODE RAW
+        ,likerock BOOLEAN   -- ENCODE RAW
+        ,likevegas BOOLEAN   -- ENCODE RAW
+        ,likebroadway BOOLEAN   -- ENCODE RAW
+        ,likemusicals BOOLEAN   -- ENCODE RAW
+    )
+
+    -- DISTSTYLE KEY
+     -- DISTKEY (userid)
+     -- SORTKEY (
+        -- userid
+        -- )
+    ;
+    -- ALTER TABLE tickit.users owner to "admin";
+
+    -- Permissions
+
+    -- GRANT RULE, SELECT, REFERENCES, TRIGGER ON TABLE tickit.users TO "admin";
+    -- GRANT SELECT ON TABLE tickit.users TO public;
+    ```
+
+
+### Create the database in Hydra
+ 1) Create a schema if you haven’t already done so
+ 2) Using that schema, run the modified DDL sql from the step above
+
+### Create the foreign tables to access the S3 parquet
+ 1) Access to S3 through parquet_s3_fdw
+
+    ```sql
+    CREATE EXTENSION parquet_s3_fdw;
+
+    CREATE SERVER parquet_s3_srv FOREIGN DATA WRAPPER parquet_s3_fdw OPTIONS (aws_region 'us-east-1');
+
+    CREATE USER MAPPING FOR CURRENT_USER SERVER parquet_s3_srv OPTIONS (user '/ACCESSKEY/', password '/SECRETKEY/');
+    Modify the CREATE TABLE statements for the foreign tables (example)
+    CREATE foreign TABLE
+    ```
+ 2) Simplify the data types
+  - All INTs and BOOLEAN to ‘int’
+    If you want to get tricky you can shorten or lengthen the ints to different sized ints, that won’t really do much except make more data available in memory cache at any given time. 
+    You will need to re-convert the 0-1 boolean ints back to boolean, we have a nice trick for doing that during the INSERT
+  - DATE and TIMESTAMP can be left alone
+  - All VARCHAR and other strings to simply ‘text’
+
+Example output:
+  
+  ```sql
+  CREATE foreign TABLE tickit.fdate
+  (
+        dateid int -- SMALLINT NOT NULL  -- ENCODE RAW
+        ,caldate DATE NOT NULL  -- ENCODE az64
+        ,"day" text -- CHAR(3) NOT NULL  -- ENCODE lzo
+        ,week int --SMALLINT NOT NULL  -- ENCODE az64
+        ,"month" text --CHAR(5) NOT NULL  -- ENCODE lzo
+        ,qtr text -- CHAR(5) NOT NULL  -- ENCODE lzo
+        ,"year" int --SMALLINT NOT NULL  -- ENCODE az64
+        ,holiday int --BOOLEAN  DEFAULT false -- ENCODE RAW
+  )
+  SERVER parquet_s3_srv
+  OPTIONS (
+    filename 's3://tiamath/tickit/date_000.parquet'
+  );
+  ```
+
+ 3) Insert the data from the foreign tables to Hydra
+ Example insert into statement, and for the most part Hydra will match the field names and convert the types just fine
+
+  ```sql 
+  INSERT INTO tickit.category
+  SELECT * FROM tickit.fcategory ;
+  ```
+
+ In case there is any type casting that needs to be done, you will need to explicitly call out the fields. Note the conversion from int to boolean in this example
+
+  ```sql
+  INSERT INTO tickit.date (dateid, caldate, "day", week, "month", qtr, "year", holiday)
+  SELECT dateid, caldate, "day", week, "month", qtr, "year", holiday != 0
+  FROM tickit.fdate ;
+  ```
```

Comment:
```suggestion
 In case there is any type casting that needs to be done, you will need to explicitly convert the fields. For example, below we convert from int to boolean:

    ```sql
    INSERT INTO tickit.date (dateid, caldate, "day", week, "month", qtr, "year", holiday)
    SELECT dateid, caldate, "day", week, "month", qtr, "year", holiday != 0
    FROM tickit.fdate ;
    ```
```